lib/diaspora_federation/entities/account_migration.rb

```diff
@@ -44,30 +65,44 @@ def self.from_hash(*args)
 
       private
 
-      # @see DiasporaFederation::Entities::Signable#signature_data
-      def signature_data
-        to_s
+      def old_identity_prop
```

Comment:
I don't know if we need this, this is private anyway, so I would just use `@old_identity` directly internally.